src/main/resources/assets/mmorpg/lang/ru_ru.json

```
@@ -383,981 +211,204 @@
+"_comment": " [CATEGORY]: CHAT MESSAGES - NAMES",
 
-	"mmorpg.rarity..common": "Обычный",
-	"mmorpg.rarity..epic": "Епический",
-	"mmorpg.rarity..legendary": "Легендарный",
-	"mmorpg.rarity..mythic": "Мистический",
-	"mmorpg.rarity..rare": "Редкий",
-	"mmorpg.rarity..runeword": "Слово руны",
-	"mmorpg.rarity..uncommon": "Необычный",
-	"mmorpg.rarity..unique": "Уникальный",
+	"mmorpg.chat.already_casting": "Уже производится заклинание",
+	"mmorpg.chat.alt_to_show_other_spell": "[Alt], чтобы показать другое заклинание",
+	"mmorpg.chat.any_item": "Любое оружие может быть использовано.",
+	"mmorpg.chat.cant_equip_that_many_supports": "Вы не можете экипировать столько поддерживающих самоцветов! Вы можете увеличить количество слотов, повысив уровень навыка.",
+	"mmorpg.chat.cant_have_more_gems_than_runes_in_runeword": "Предметы с руными словами не могут иметь больше самоцветов, чем рун, вставленных в них",
+	"mmorpg.chat.cant_use_multiple_same_supports": "Нельзя использовать несколько одинаковых поддерживающих самоцветов.",
+	"mmorpg.chat.cast_failed": "[Не удалось произвести заклинание]: ",
+	"mmorpg.chat.character_load_info": "НАЖМИТЕ, чтобы загрузить этого персонажа\n\nСохраненные характеристики персонажа:\n- Ваш уровень и опыт\n- Ваши заклинания, статистика, таланты, настройка панели действий\n\nНе сохраняются ваша экипировка, самоцветы, драгоценные камни и т. д.\n\nКаждый персонаж начинает с 1 уровня",
+	"mmorpg.chat.command_block_unavaliable": "Командные блоки отключены, это препятствует игре в Подземелья Mine and Slash!",
+	"mmorpg.chat.create_new_character": "Создать нового персонажа\n\nИмя должно быть уникальным",
+	"mmorpg.chat.enemy_too_close": "Вы не можете телепортироваться, когда рядом враги.",
+	"mmorpg.chat.gear_drop": "Вы не соответствуете требованиям этого предмета.",
+	"mmorpg.chat.gem_socketed": "Самоцвет вставлен",
+	"mmorpg.chat.how_to_enable_command_block": "Чтобы включить, перейдите в файл server.properties и установите enable-command-block на true.",
+	"mmorpg.chat.lack_aura_capacity": "У вас недостаточно емкости ауры, чтобы экипировать все эти ауры.",
+	"mmorpg.chat.level_up_message_down": "%1$s Уровень: %2$s > %3$s!",
+	"mmorpg.chat.level_up_message_up": "Повышение уровня!",
+	"mmorpg.chat.max_map_rarity": "Эта карта имеет максимальную возможную редкость.",
+	"mmorpg.chat.max_map_rarity_for_lvl": "Эта карта имеет максимальную возможную редкость для своего уровня.",
+	"mmorpg.chat.no_charges": "У вас нет зарядов для заклинаний.",
+	"mmorpg.chat.no_duplicate_aura": "Вы не можете экипировать одинаковые самоцветы ауры.",
+	"mmorpg.chat.no_mana": "Недостаточно ресурсов для произнесения.",
+	"mmorpg.chat.none_mage": "Оружие для Не-Магов может быть использовано.",
+	"mmorpg.chat.not_in_this_dimension": "Вы не можете произнести это заклинание в этом измерении.",
+	"mmorpg.chat.not_map": "Это не карта.",
+	"mmorpg.chat.not_mns_weapon": "Вам нужно оружие Mine and Slash (Обладающее статистикой), чтобы произнести заклинания.",
```

Если я правильно понял, тут имеется ввиду, что нужно оружие из MnS, а его можно получить только после применения души снаряжения на любую вещь
```suggestion
	"mmorpg.chat.not_mns_weapon": "Вам нужно оружие Mine and Slash (Преобразовано душой), чтобы произнести заклинания.",
```

---

изменено